boilerplate/openshift/custom-catalog-osd-operator/custom-catalog-build-push.sh

```diff
@@ -36,12 +36,12 @@ function build_catalog_image() {
 
   # preserve logging output from command by duplicating the file descriptor
   exec 5>&1
-  BUILD=$(${opm_local_executable} index add \
+  BUILD="$(${opm_local_executable} index add \
```

Comment:
so what I meant was quoting `"$BUILD"` when you're going to echo it to grep. Viz:

```
[efried@efried hive]$ x=`printf 'foo\nbar\nfoo\nbaz\n'`
[efried@efried hive]$ echo $x
foo bar foo baz
[efried@efried hive]$ echo $x | grep -c foo
1
[efried@efried hive]$ echo "$x"
foo
bar
foo
baz
[efried@efried hive]$ echo "$x" | grep -c foo
2
```

Reply:
Ahhh that makes more sense, I didn't see what you were talking about but assumed you knew what was up haha. Got this fixed. Thanks.